src/SearchResultPage/searchResults.js

```diff
@@ -949,62 +468,68 @@ export function searchResults() {
                                 </div>
                            
                         </li>`;
-          }
-        })
-        .join('')}
+            }
+          })
+          .join('')}
     `;
   };
-
   const connectedHitsWithInjectedContent = connectHits(renderHits);
 
+  const widgets = {
+    category: {
+      widget: refinementList({
+        container: document.querySelector('#dynamic-widgets'),
+        attribute: 'category',
+      }),
+      container: document.createElement('div'),
+    },
+    // custom: {
+    //   widget: customWidget({container: document.querySelector('#dynamic-widgets')}),
+    //   container: document.createElement('div')
+    // }
+  };
+  const renderDynamicWidget = (renderOptions, isFirstRender) => {
+    console.log(renderOptions);
+  };
+
+  const customDynamicWidgets = EXPERIMENTAL_connectDynamicWidgets(
+    renderDynamicWidget
+  );
+
   search.addWidgets([
+    customQueryRuleCustomData({
+      container: document.querySelector('#banner'),
+    }),
+    customCurrentRefinements({
+      container: document.querySelector('#current-refinements'),
+    }),
     customConfigure({
       container: document.querySelector('#configure'),
```

Comment:
unrelated, but why is a configure displayed?

Reply:
Maybe @HugoWith knows